index.js

```diff
@@ -8,11 +8,100 @@
 'use strict';
 
 var fs = require('fs');
+var path = require('path');
+
+/**
+ * Check if the given `filepath` exists.
+ *
+ * ```js
+ * var res = exists('package.json');
+ * console.log(res);
+ * //=> "package.json"
+ *
+ * var res = exists('fake-file.json');
+ * console.log(res)
+ * //=> false
+ * ```
+ *
+ * @name exists
+ * @param  {String} `filepath` filepath to check for.
+ * @return {String|Boolean} Returns the found filepath if it exists, otherwise returns `false`.
```

Comment:
@doowb suggested we make `fs-resolve-file` or something like that to do this. that seems like a good idea. that way we can preserve the boolean behavior of this lib in all cases, instead of making it optional. sound ok to you?